packages/modules/manage-versions/src/components/ManageVersions/ChangesTab/ChangesTab.tsx

```diff
@@ -20,6 +20,7 @@ export type ChangesTabProps = {
   loadMoreChanges: () => void;
   onVersionCreated: () => void;
   latestVersion: NamedVersion | undefined;
+  shouldShowChangedFiles?: boolean;
```

Comment:
I would say this flag is not really needed. This new column should be beneficial to everyone.

Reply:
removed the flag